dashboard/lib/api/v1/school_autocomplete.rb

```diff
@@ -12,7 +12,8 @@ def self.get_matches(query, limit, use_new_search)
       terms = get_query_terms query
       match_terms = []
       terms.each do |term|
-        match_terms.push "CASE when (MATCH(name, city) AGAINST('#{term}' IN BOOLEAN MODE) OR zip = '#{term}') THEN 1 ELSE 0 END"
+        # sanitize_sql_for_order won't be a public method until Rails 5.2
+        match_terms.push ActiveRecord::Base.send(:sanitize_sql_for_order, ["CASE when (MATCH(name, city) AGAINST(? IN BOOLEAN MODE) OR zip = ?) THEN 1 ELSE 0 END", term, term])
```

Comment:
I feel much better about this with your comment that this is becoming a public method.